kb/plane_geometry/formal_statements/statements/statements_scs/stat_triangle_cos_angle_side_opposite.scs

```diff
@@ -0,0 +1,14 @@
+stat_triangle_cos_angle_side_opposite <- statement;;
+
+stat_triangle_cos_angle_side_opposite -> rrel_key_sc_element: concept_triangle; nrel_cos; concept_angle; nrel_side; nrel_opposite;;
+
+stat_triangle_cos_angle_side_opposite => nrel_main_idtf: [Акс.(треугольник;косинус*;угол;сторона*;быть противолежащим*)] (*<- lang_ru;;*);;
+stat_triangle_cos_angle_side_opposite => nrel_main_idtf: [Ax.(triangle;cos*;angle;side*;opposite*)] (*<- lang_en;;*);;

```

Comment:
It isn't an axiom. It is a statement. You should use [Stat.(...)] insted of [Ax.(...)]